src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -136,7 +136,7 @@ <h2 class="card-pf-title">
                             *ngIf="isRuntimeDisabled(runtime) === true">
                         Contribute <i class="pficon pficon-info"></i>
                       </span>
-                      <span class="f8launcher-tags-label" container="body" triggers="click"
+                      <span class="f8launcher-tags-label f8launcher-suggested-tag-label" container="body" triggers="click"
```

Comment:
Although the "suggested-tag" selector has been renamed to f8launcher-suggested-tag-label, it's still an empty/dummy selector. Considering this selector doesn't add/change the styling in the page, it should be removed.

My suggestion is to simply rename f8launcher-tags-label as f8launcher-suggested-tag-label, f8launcher-prerequisite-tag-label, and f8launcher-contribute-tag-label.

For example:

<span class="f8launcher-tags-label contribute" ...
<span class="f8launcher-tags-label f8launcher-suggested-tag-label" ...
<span class="f8launcher-tags-label prerequisite" ...

Would become:

<span class="f8launcher-contribute-tag-label" ...
<span class="f8launcher-suggested-tag-label" ...
<span class="f8launcher-prerequisite-tag-label" ...

Then there are no empty/dummy selectors added only for the unit test. Each selector contributes some styling to the page.

@mindreeper2420 Does this sound like a reasonable approach?

Reply:
@dlabrecq agreed - a class should not be empty or just a placeholder for something to catch. If it can be set to be multi-purpose (styling and a test resource), that would be ideal.